rasa/core/tracker_store.py

```diff
@@ -1257,6 +1349,14 @@ def _create_from_endpoint_config(
         tracker_store = DynamoTrackerStore(
             domain=domain, event_broker=event_broker, **endpoint_config.kwargs
         )
+    elif endpoint_config.type.lower() == "firebase-realtime":
+        tracker_store = FirebaseRealtimeTrackerStore(
+            database_url=endpoint_config.url,
```

Comment:
Please maintain the order of args, first should be domain, followed by `database_url`, then `event_broker` etc.